samples/model-builder/predict_image_classification_sample.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import base64
+
+from typing import List
+
+from google.cloud import aiplatform
+
+import tensorflow as tf
+
+
+#  [START aiplatform_sdk_predict_image_classification_sample]
+def predict_tabular_classification_sample(
```

Comment:
Should this read `predict_image_classification_sample`?

Reply:
yes, that looks like a typo.

Reply:
I fixed it.